migration-engine/migration-engine-tests/tests/migrations/basic/vitess.rs

```diff
@@ -6,20 +6,20 @@ fn reordering_and_altering_models_at_the_same_time_works(api: TestApi) {
         model A {
             id Int @id
             name Int @unique
-            c C @relation(name: "atoc", fields: [name], references: [name], onUpdate: Restrict)
+            c C @relation(name: "atoc", fields: [name], references: [name], onDelete: Restrict)
```

Comment:
Why was there no `onDelete` previously?

Reply:
We just went with defaults. Now we don't allow anything else than `onUpdate: NoAction`, so needed to do something here to get the tests passing.

Reply:
Ah I see, makes sense.